### ISSUE TITLE
Brave Today image requests should omit certain headers

### ISSUE THREAD

**Issue by fmarier** [brave/brave-browser#12684]
Every application using our private CDN is asked to remove the following headers from their requests:
- `Accept-Language`
- `Cookie`
- `DNT`
- `Referer`
- `User-Agent`

I suspect we already omit the `Cookie` header when loading images in Brave Today.

**Comment by petemill** [comment]
These are the current request headers made when we request an item

![image](https://user-images.githubusercontent.com/741836/100274452-4c8b1700-2f13-11eb-93ef-9850cb2c0bd5.png)

Looks like we just need to remove

- `user-agent`
- `accept-language`

Is that right @fmarier ?

**Comment by petemill** [comment]
With generating the request via JS `fetch`, [it's not currently possible to customize the `user-agent` header (even though it should be).](https://bugs.chromium.org/p/chromium/issues/detail?id=571722).

**Comment by fmarier** [comment]
> Looks like we just need to remove
> 
> * `user-agent`
> * `accept-language`

Yes, those two, and potentially DNT though you'd only see that one if you enabled it in `brave://settings/cookies`:
![Screenshot from 2020-11-25 13-05-49](https://user-images.githubusercontent.com/167821/100281541-f623d580-2f1e-11eb-94ce-9ff13fd5dbef.png)

**Comment by petemill** [comment]
Going to re-open this and leave it open since the `user-agent` and `dnt` headers are still being transmitted. Unavoidable with the current fetch API in chromium until we can move to C++

**Comment by petemill** [comment]
@fmarier are you ok with moving this to a p3 since there should not be that many different buckets of users if we're only sending UA and dnt?

**Comment by fmarier** [comment]
P3 sounds fine. Let's not forget about it, but it doesn't need to be urgently fixed given we're not leaking very much info anymore.

**Comment by mattmcalister** [comment]
@petemill is this addressable now? I'd like to close out old issues, if possible.
